x-pack/plugins/ingest_pipelines/public/application/services/documentation.ts

```diff
@@ -23,6 +29,12 @@ export class DocumentationService {
     this.processorsUrl = `${links.ingest.processors}`;
     this.handlingFailureUrl = `${links.ingest.pipelineFailure}`;
     this.putPipelineApiUrl = `${links.apis.createPipeline}`;
+    this.simulatePipelineApiUrl = `${links.apis.simulatePipeline}`;
+    this.enrichDataUrl = `${links.ingest.enrich}`;
+    this.enrichDataUrl = `${links.ingest.geoMatch}`;
```

Comment:
Looks like this should be `this.geoMatchUrl`.

Reply:
Thanks for catching this!